package.json

```diff
@@ -32,12 +49,13 @@
   "license": "MIT",
   "devDependencies": {
     "add-event-handler": "^1.0.0",
-    "babel": "^5.8.21",
+    "babel-plugin-transform-class-properties": "^6.3.13",
+    "babel-preset-es2015": "^6.3.13",
+    "babel-register": "^6.3.13",
```

Comment:
Also, using `babel-register` means that we can use es2015 in the test module, itself.